test/e2e/tests/01-required-and-optional.js

```diff
@@ -0,0 +1,261 @@
+      .setValue('select[name="country"]', 'USA')
+      .setValue('select[name="state"]', 'IL')
+      .setValue('input[name="zip"]', '60603')
+      .click('.form-panel .usa-button-primary');
+    expectNavigateAwayFrom(client, '/veteran-information/veteran-address');
+
+    // Contact Information Page.
+    client.expect.element('input[name="email"]').to.be.visible;
+    client
+      .setValue('input[name="email"]', 'bills@bard.com')
+      .setValue('.input-section > div:nth-child(2) input', 'bills@bard.com')
+      .setValue('input[name="phone"]', '3125551212')
+      .setValue('.input-section > div:nth-child(4) input', '3125551213')
+      .click('.form-panel .usa-button-primary');
+    expectNavigateAwayFrom(client, '/veteran-information/contact-information');
+
+    // Military Service Information Page.
+    client.expect.element('select[name="lastServiceBranch"]').to.be.visible;
+    client
+      .setValue('select[name="lastServiceBranch"]', 'army')
+      .setValue('select[name="lastEntryMonth"]', 'Oct')
+      .setValue('select[name="lastEntryDay"]', '10')
+      .setValue('input[name="lastEntryYear"]', '2000')
+      .setValue('select[name="lastDischargeMonth"]', 'Nov')
+      .setValue('select[name="lastDischargeDay"]', '11')
+      .setValue('input[name="lastDischargeYear"]', '2004')
+      .setValue('select[name="dischargeType"]', 'honorable')
+      .click('.form-panel .usa-button-primary');
+    expectNavigateAwayFrom(client, '/military-service/service-information');
+
+    // Military Service Additional Information Page.
+    client.expect.element('input[name="purpleHeartRecipient"] + label').to.be.visible;
+    client
+      .click('input[name="purpleHeartRecipient"] + label')
+      .click('input[name="isFormerPow"] + label')
+      .click('input[name="postNov111998Combat"] + label')
+      .click('input[name="disabledInLineOfDuty"] + label')
+      .click('input[name="swAsiaCombat"] + label')
+      .click('input[name="vietnamService"] + label')
+      .click('input[name="exposedToRadiation"] + label')
+      .click('input[name="radiumTreatments"] + label')
+      .click('input[name="campLejeune"] + label')
+      .click('.form-panel .usa-button-primary');
+    expectNavigateAwayFrom(client, '/military-service/additional-information');
+
+    // VA Benefits Basic Info page.
+    client.expect.element('input[name="compensableVaServiceConnected-0"] + label').to.be.visible;
+    client
+      .click('input[name="compensableVaServiceConnected-0"] + label')
+      .click('input[name="isVaServiceConnected-0"] + label')
+      .click('input[name="receivesVaPension-0"] + label')
+      .click('.form-panel .usa-button-primary');
+    expectNavigateAwayFrom(client, '/va-benefits/basic-information');
+
+    // Financial disclosure page.
+    client.expect.element('input[name="provideFinancialInfo-0"] + label').to.be.visible;
+    client
+      .click('input[name="provideFinancialInfo-0"] + label')
```

Comment:
I wonder if for this one we should instead click `input[name="provideFinancialInfo-1"] + label` because that will expand a second question that we should also click a response for.